lib/product-import.coffee

```diff
@@ -417,6 +418,8 @@ class ProductImport
       .then (existingProducts) =>
         existingProducts or @_getExistingProductsForSkus(@_extractUniqueSkus([prodToProcess]))
       .then (existingProducts) =>
+        if (@beforeCreateOrUpdateCallback)
```

Comment:
Hi @lojzatran .. it would be great if this callback fn can be also async so people won't hit unexpected behaviour by running async code

Reply:
https://github.com/sphereio/sphere-product-import/pull/166/commits/bc949ceeae4c07d419f3245b624649aecb12c4a0